tests/core/test_disk_d88.c

```
@@ -0,0 +1,149 @@
+		BYTE sect_hdr[D88_SECTOR_HDR_SIZE];
+		BYTE *data = calloc(1, sector_size);
+
+		memset(sect_hdr, 0, sizeof(sect_hdr));
+		sect_hdr[0] = 0;                  /* c */
+		sect_hdr[1] = 0;                  /* h */
+		sect_hdr[2] = (BYTE)(s + 1);      /* r */
+		sect_hdr[3] = 1;                  /* n (256 bytes) */
+		memcpy(sect_hdr + 4, &sectors, 2);     /* sectors per track */
+		memcpy(sect_hdr + 14, &sector_size, 2);/* sector size */
+		fwrite(sect_hdr, 1, sizeof(sect_hdr), fp);
+		fwrite(data, 1, sector_size, fp);
```

Cppcheck (2.20.0)

[warning] If memory allocation fails, then there is a possible null pointer dereference

(nullPointerOutOfMemory)